src/freenas/etc/netcli

```diff
@@ -250,9 +250,9 @@ def configure_interface():
             continue
 
     while True:
-        _input = input(_("Reset network configuration? (y/n) ")).lower()
+        _input = input(_("Restore this interface to saved configuration? (This causes a momentary disconnection of the network.) (y/n) ")).lower()
```

Comment:
What saved configuration are you referring to?

I think this is a step back since its actually resetting, all interface values are being blanked.

Reply:
This was from input from other people.  The problem is that "reset" is very ambiguous.
But I see what you mean, will update.